### ISSUE TITLE
Error Installing Zen Daemon

### ISSUE THREAD

**Issue by elkimek** [ZencashOfficial/Sphere_by_Horizen#2]
If switched to fullnode, red error on the left side appears "Error Installing Zend Daemon".

OS: W10 1803 (17134.471)
PS ver:  5.1.17134.407
Logs:

```
[2018-12-17T15:55:12.0239Z] [info] zend not found.
[2018-12-17T15:55:12.0240Z] [info] installing zend...
[2018-12-17T15:55:12.0240Z] [info] get-zend-directory called C:\Users\xx\xx\AppData\Roaming/horizen/
[2018-12-17T15:55:12.0266Z] [info] zend not found.
[2018-12-17T15:55:12.0271Z] [info] write-zend-package called
[2018-12-17T15:55:12.0287Z] [info] write-zend-package-success: [object Object]
[2018-12-17T15:55:12.0635Z] [error] Exception calling "ExtractToDirectory" with "2" argument(s): "��st cesty C:\Users\xx\xx\AppData\Roaming\horizen\zend-
temp.zip nebyla nalezena."
At line:1 char:52
+ ... FileSystem';[IO.Compression.ZipFile]::ExtractToDirectory('C:/Users/xx ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : DirectoryNotFoundException
```

**Comment by al73rna** [comment]
Please make sure you have the .net framework 4.5 installed. if not, please install it and try again.
If it is installed and you're still getting the error, please try running these lines in your powershell prompt and try again:
```
Add-Type -assembly "System.IO.Compression"
Add-Type -assembly "System.IO.Compression.Filesystem"
```

the problem is that we are relying on powershell to extract the binaries from zipfiles which has some issues of it's own. we will switch to a different solution if this becomes an issue for more people.

**Comment by elkimek** [comment]
Did not help. I also tried to extract the files directly to /horizen/zend/ but still getting the same error.

**Comment by ahillelt** [comment]
Just wanted to confirm, experiencing same issue on Windows 10, with 4.5 installed and running the commands suggest by @al73rna in powershell. ZEN Daemon cannot install, log is similar to @elkimek

**Comment by elkimek** [comment]
Issue persist in new release. Advice?

**Comment by ahillelt** [comment]
Anyone knows what status is of this bug, I think there are quite a few of us that cannot use Sphere at the moment on Windows 10.

**Comment by treepi3** [comment]
I'm experiencing a Zen daemon installation failure on Linux as well. Solus 4.0 Budgie to be exact. I need to full node for privacy too. :(

**Comment by vherus** [comment]
Hi there

Please accept my apologies for the lack of activity on this report!

Though version 1.1.0-beta fixed many issues, we're aware that the zend installation & blockchain sync process can still be a little tough to deal with in some situations. This is a super important issue to us, it shouldn't be as difficult as it currently is to sync up the wallets node. In figuring out a solution we are taking every care to ensure we fix this with a high degree of Horizen polish, with the end goal being a swifter, easy to use installation & sync process.

Please bare with us as we tackle issues like this during beta phase of the software, and thank you for your support.

**Comment by Sto1cNate** [comment]
@elkimek 
This issue was resolved in a hot fix from 1.1.1-beta
https://github.com/ZencashOfficial/Sphere_by_Horizen/releases/tag/desktop-v1.1.1-beta

@zennermac this issue should be able to be closed.
